### DISCUSSION TITLE
convert other formats suggestion option

### DISCUSSION THREAD

**Discussion by FrostKnight** [vkbo/novelWriter#1431] [Ideas]
Such as the one for manuskript, for example and others, ones that don't change often preferably of course.

Otherwise it would be a headache.

I looked up to see if anything similar existed, but couldn't clearly say whether anything existed similar. I tried typing in, import and format but didn't see anything similar, just a heads up.

**Comment by vkbo** [comment]
There is a converter for yWriter, but none other that I know of. novelWriter itself will only support standard formats for now, and then preferably open formats. I plan to add an export to a file hierarchy of Markdown, so any other app that can import that should be able to import the project too. novelWriter has the ability to import a novel as Markdown or plain text (the latter requires some editing) and then split it into a hierarchy of files. I've done this myself with a few public domain novels that I use as test data. It's a bit of work, but not too much. So, basically, getting text in or out of novelWriter isn't very difficult, but it depends on the software in the other end too. At least on my end, I try to avoid getting people stuck in some quirky locked in format.
